Makefile

```diff
@@ -1,15 +1,51 @@
-QA_DOCKER_IMAGE=jakzal/phpqa:alpine
+QA_DOCKER_IMAGE=rollerworks/search-phpqa:latest
 QA_DOCKER_COMMAND=docker run -it --rm -v "$(shell pwd):/project" -w /project ${QA_DOCKER_IMAGE}
-ELASTICSEARCH_DOCKER_COMPOSE=lib/Elasticsearch/docker-compose.yml
 
-dist: cs-full phpstan test-full
+dist: install cs-full phpstan test-full
```

Comment:
Install probably not needed here, especially since there's no `composer.phar` on all systems (mine, for example).